src/helpers/Utils/Utils.js

```diff
@@ -187,6 +187,24 @@ export const normalizeLanguageName = (language) => {
   return language;
 };
 
+/**
+ * Create an object with properties for each object with an `id` in an array
+ * @function arrayWIdsToObject
+ * @param {Array} An array of objects, each has an `id` property with a string value
+ * @returns {Object} Oobject with property for each array object keyed by the `id`
+ */
+export function arrayWIdsToObject(arrayOfObjs) {
```

Comment:
I think this can be a helper inside the actions reducer, I don't see its utility exposed as a Volto global helper.

Reply:
I put it in `Utils` because there are many cases in CMF's and Plone's data shapes where there are arrays of objects with `id` properties.  So the function isn't specific to actions and there's likely to be fruitful usage outside of actions.

Reply:
If it's a global helper, it needs a better name, otherwise it's hard to remember.

Reply:
Such as?

Reply:
None of those seem to be clear improvements to me.  If any of them get an up-vote from anyone else I'll make the change just to finish this discussion.  Until then I'm resolving this.